apps/web/components/organisms/AuthModal/AuthModal.tsx

```diff
@@ -0,0 +1,344 @@
+"use client";
+
+import {
+  Modal,
+  Stack,
+  Text,
+  Button,
+  TextInput,
+  PasswordInput,
+  Divider,
+  Group,
+  Alert,
+  Anchor,
+} from "@mantine/core";
+import { IconBrandGoogle, IconInfoCircle } from "@tabler/icons-react";
+import { useState, useEffect } from "react";
+import { useForm } from "react-hook-form";
```

Comment:
We are using mantine and they have their own mantine/form library (https://mantine.dev/form/package/)
Do you mind refactoring it to use this please?